tests/cases/fourslash/server/declarationMapsEnableMapping_NoInline.ts

```diff
@@ -1,3 +1,89 @@
+<<<<<<< HEAD
```

Comment:
VSCode did not detect this as a merge conflict for some reason. I'll post a fix shortly.